modules/admin_manual/pages/oidc/index.adoc

```diff
@@ -0,0 +1,162 @@
+= What is OpenID Connect (OIDC)
+:toc:
+:panva-node-oidc-provider-url: https://github.com/panva/node-oidc-provider
+:openid-connect-frontchannel-logout-url: https://openid.net/specs/openid-connect-frontchannel-1_0.html
+
+According to openid.net/connect:
```

Comment:
I would add here a introduction section as we do in all other documents making it better readble
```
== Introduction
According to openid.net/connect:
```

Reply:
On reviewing that section, removing line 6 and just leaving the cited paragraph would be equally as helpful.

Reply:
Actually, I like your suggestion better.